internal/npm_install/npm_install.bzl

```diff
@@ -26,6 +26,11 @@ load("//internal/common:os_name.bzl", "is_windows_os")
 load("//internal/node:node_labels.bzl", "get_node_label", "get_npm_label", "get_yarn_label")
 
 COMMON_ATTRIBUTES = dict(dict(), **{
+    "timeout": attr.int(
```

Comment:
I'm confused I thought this was being removed

Reply:
This along with quiet gets passed down to the rctx.execute function:
``` 
repository_ctx.report_progress("Running npm install on %s" % repository_ctx.attr.package_json)
    result = repository_ctx.execute(
        [repository_ctx.path("npm.cmd" if is_windows_host else "npm")],
        timeout = repository_ctx.attr.timeout,
        quiet = repository_ctx.attr.quiet,
    )
```
so its not an argument passed to yarn or npm